extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,243 @@
+}
+
+.i-amphtml-story-dev-tools-device-chips .i-amphtml-story-dev-tools-device-chip[inactive] {
+  max-width: 0px !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive] {
+  background: transparent !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive]:hover {
+  background: #1d1d1d !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive] svg {
+  transform: rotate(90deg) !important;
+  background: #ffffff22 !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip:not([inactive]):not(.i-amphtml-story-dev-tools-add-device) .i-amphtml-story-dev-tools-device-chip-add-stick {
+  fill: transparent !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device {
+  background: #79B3FF !important;
+  color: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device svg {
+  fill: #79B3FF !important;
+  background: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device:hover {
+  background: #92c1ff !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-bg {
+  position: fixed !important;
+  left: 0 !important;
+  top: 0 !important;
+  height: 100% !important;
+  width: 100% !important;
+  background: transparent !important;
+  display: flex !important;
+  justify-content: center !important;
+  align-items: center !important;
+  z-index: 1 !important;
+  transition: background 0.2s ease-in-out !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-container {
+  background: black !important;
+  width: min(650px, 90%) !important;
+  border-radius: 10px !important;
+  padding: 40px !important;
+  position: relative !important;
+  transition: transform 0.2s ease-in-out, opacity 0.2s ease-in-out !important;
+  transform: translateY(3vh) !important;
+  opacity: 0 !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-bg[active] {
+  background: #111111cc !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-bg[active] .i-amphtml-story-dev-tools-device-popup-container  {
+  transform: translateY(0) !important;
+  opacity: 1 !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-container h1 {
+  font-size: 12px !important;
+  text-transform: uppercase !important;
+  opacity: .4 !important;
+  letter-spacing: 1px !important;
+  font-weight: normal !important;
+  margin: 0px !important;
+  margin-bottom: 5px !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-container h1:not(:first-child) {
+  margin-top: 25px !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-container.i-amphtml-story-dev-tools-device-popup-add-devices h1 {
+  margin-left: 15px !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-container a {
+  color: #ffffffcc !important;
+  text-decoration: none !important;
+  display: block !important;
+  padding: 10px 0 !important;
+  transition: color 0.2s ease-in-out !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-container a svg {
```

Comment:
As much as possible, use classes over tagNames. Most advanced production systems would forbid using any tagName into the CSS, as relying on the DOM structure too fragile and can be easily broken.